demo-ehr-app/README.md

```diff
@@ -0,0 +1,30 @@
+# React + TypeScript + Vite
```

Comment:
README should updated by including the details about the demo-ehr-app and how to set it up.

Reply:
Updated. https://github.com/wso2/solutions-healthcare-demos/pull/2/commits/2c489a67c1aa2e1022389842e01410f0410ea060